scipy/special/basic.py

```diff
@@ -1394,15 +1388,14 @@ def lpmn(m, n, z):
     if (m < 0):
         mp = -m
         mf, nf = mgrid[0:mp+1, 0:n+1]
-        sv = errprint(0)
-        if abs(z) < 1:
-            # Ferrer function; DLMF 14.9.3
-            fixarr = where(mf > nf, 0.0,
-                           (-1)**mf * gamma(nf-mf+1) / gamma(nf+mf+1))
-        else:
-            # Match to clpmn; DLMF 14.9.13
-            fixarr = where(mf > nf, 0.0, gamma(nf-mf+1) / gamma(nf+mf+1))
-        sv = errprint(sv)
+        with ufuncs.errstate(all='ignore'):
```

Comment:
OK, I looked at git blame a bit down to 
``` 
a6270c97 Lib/special/basic.py   (Travis Oliphant     2005-10-06 03:09:02 +0000   6) from _cephes import *
```

then on to
```
f41c1dae scipy/special/basic.py (Warren Weckesser    2010-11-20 16:49:57 +0000   8) from _cephes import ellipk, mathieu_a, mathieu_b, iv, jv, gamma, psi, zeta, \
```

and 
```
da453dca scipy/special/basic.py (Martin Teichmann    2011-08-11 09:17:31 +0200   8) from _cephes import ellipkm1, mathieu_a, mathieu_b, iv, jv, gamma, psi, zeta, \
```

and then `_cephes` becomes `_ufuncs`:
```
08b56a03 scipy/special/basic.py (Pauli Virtanen      2012-10-07 04:07:09 +0300   9) from _ufuncs import ellipkm1, mathieu_a, mathieu_b, iv, jv, gamma, psi, zeta, \
```

followed by the Python 3 transition where the import becomes `from ._ufuncs import ...` also in 2012.

So yeah, it has always been like that :-), and I'm not sure there's much point messing with it.  Scratch my comment above :-).